media/js/firefox/new/scene1.js

```diff
@@ -60,7 +60,8 @@
             if (client.isFirefoxDesktop) {
                 client.getFirefoxDetails(function(data) {
                     // data.accurate will only be true if UITour API is working.
-                    if (data.channel === 'release' && data.isUpToDate && data.accurate) {
+                    // bug 1301721 only use major Firefox version until 49.0 is released
+                    if (data.channel === 'release' && client._isFirefoxUpToDate(false) && data.accurate) {
```

Comment:
I think calling it again here explicitly is fine, as it also serves as a reminder that we need to change it back again in the future.